### ISSUE TITLE
`s3_methods_generic()` gives bad `class` column

### ISSUE THREAD

**Issue by dongzhuoer** [hadley/S3#19]
As in your book https://adv-r.hadley.nz/s3.html#finding-methods

![image](https://user-images.githubusercontent.com/15871952/50903482-e7ca8e80-1458-11e9-84bf-6f9b23ac6031.png)

**Comment by dongzhuoer** [comment]
Not what the readme promise

```r
s3_methods_generic("summary")
#> # A tibble: 35 x 4
#>    generic class                 visible source             
#>    <chr>   <chr>                 <lgl>   <chr>              
#>  1 summary aov                   TRUE    stats              
#>  2 summary aovlist               FALSE   registered S3method
#>  3 summary aspell                FALSE   registered S3method
#>  4 summary check_packages_in_dir FALSE   registered S3method
#>  5 summary connection            TRUE    base               
#>  6 summary data.frame            TRUE    base               
#>  7 summary Date                  TRUE    base               
#>  8 summary default               TRUE    base               
#>  9 summary ecdf                  FALSE   registered S3method
#> 10 summary factor                TRUE    base               
#> # ... with 25 more rows
```

**Comment by hadley** [comment]
Fixed in acd3f2443990835f3427da3ade46590a92b7f732
